### ISSUE TITLE
Possibility to download a bounding box and now the whole planet

### ISSUE THREAD

**Issue by carlos-mg89** [racemap/elevation-service#12]
First of all, it looks like I forgot to reply back here https://github.com/racemap/elevation-service/issues/8 . It worked like a charm and the quality of the elevation data is awesome. Thanks!

I was wondering if it'd be possible to download only a subset of the whole elevation data. I'd need Europe or even a smaller map. Is it possible or could you tell me if there's something I could do to make it possible?

Basically I want to download the elevation data for a given bounding box. That's all. Instead of the whole planet.

Thanks in advance!

**Comment by normanrz** [comment]
That is possible, but requires some manual work. You need to figure out the bounding box you're interested in, in terms of lat and lng extents. The you can download individual tiles of the data. The URL scheme is implemented here: https://github.com/racemap/elevation-service/blob/master/tileset.js#L24-L31
For example: `https://elevation-tiles-prod.s3.amazonaws.com/skadi/S01/S01W001.hgt.gz`

You would download the files in your bbox and put them in the same folder structure as the full dataset. The elevation service should be able to work with that.

**Comment by carlos-mg89** [comment]
Alright, I'm still waiting for all the files to be downloaded, but it looks like I've reached a way to download only the files I wanted. Thanks to @normanrz for all the assistance, this is the code I've managed to do:

**Comment by carlos-mg89** [comment]
Thank you again @normanrz certainly that method in tileset.js was the clue. Quite easy to figure out. I've generated a script and I'm already downloading the only necessary files for my case.

Cheers!

**Comment by carlos-mg89** [comment]
One last thing, for whoever want to use this project with a subset of the elevation data, as I did. Once you have the .gz files that you need already downloaded, it's necessary to put them in folders.

For example, if you have these .gz files:
N01E027.hgt.gz
N01E028.hgt.gz

You'll need to put them in the N01 folder.

Aside from this, when I tried to run docker I received this error:
```
docker run --rm -v/home/carlos/workspace/elevation-data:/app/data -p3000:3000 racemap/elevation-service
yarn run v1.13.0
$ micro index.js
micro: Accepting connections on port 3000
micro: Gracefully shutting down. Please wait...
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/app/data/N43/N43W002.hgt.gz'
Emitted 'error' event at:
    at lazyFs.open (internal/fs/streams.js:115:12)
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

You can easily solve it by downloading the `N43W002.hgt.gz` file and storing it on N43 folder. This way docker will be able to correctly start the elevation service on port 3000.
